skimage/filters/_unsharp_mask.py

```diff
@@ -27,7 +27,7 @@ def unsharp_mask(
 
     Parameters
     ----------
-    image : [P, ..., ]M[, N][, C] ndarray
+    image : (M[, ...][, C]) ndarray
```

Comment:
To be consistent with the changes in, say, `skimage/exposure/_adapthist.py`, shouldn't the change be:
```suggestion
    image : ndarray, shape (M[, ...][, C])
```
?

Reply:
Both of these styles are actually quite common. I would say `{shape} ndarray` is slightly more frequent (subjectively, 65% and 35%).

Reply:
happy to leave this standardisation to a later PR. 🙏